docs/src/user-manual/glossary.md

```diff
@@ -0,0 +1,368 @@
+  formats deemed more suitable for dissemination and access across space and
+  time (particularly over networked environments), based on factors such as:
+  open format specifications, smaller file sizes, widespread adoption and
+  support (particularly in web browsers), high compression rates, etc. Access
+  derivatives are typically used in the creation of **Dissemination Information
+  Packages (DIPs)**.
+
+## Directory
+
+A conceptual **object** representing a collection of files, used to provide
+human-understandable groupings of related digital objects in a filesystem.
+Directories may also contain subdirectories, and are typically organized
+hierarchically. Often used in the organization of files in a **package**, such
+as a **SIP** or **AIP**.
+
+## Dissemination Information Package (DIP)
+
+A type of **package** derived from one or more **AIPs** for delivery to an end
+user in response to an authorized access request. A DIP will typically include
+**derivative** versions of the original preservation **objects** using different
+**file formats** more suitable to access environments (i.e. lower resolutions,
+more commonly supported by web browsers, etc).
+
+The term Dissemination Information Package was originally defined in the
+[OAIS](https://en.wikipedia.org/wiki/Open_Archival_Information_System) Reference
+Model created by the Consultative Committee for Space Data Systems (and since
+formally recognized as an international standard in ISO 14721).
+
+## Document
+
+An **object** consisting of information or data fixed to some media.
+
+## Event
+
+A preservation-relevant action that involves at least one **object** and/or
+**agent**. Events are typically captured in the preservation **metadata** of a
+**package** during a preservation **workflow** using the
+[PREMIS](https://www.loc.gov/standards/premis/) metadata specification, so that
+an archival chain of custody is preserved across all **tasks** that might occur
+in a workflow.
+
+## File
+
+A named and ordered sequence of binary data, organized as a content bitstream
+that can be stored and transmitted, and typically bearing a specific **format**.
+
+## File format
+
+A standardized method of specifying how bits are
+used to encode information in a digital storage medium such as a **file**. A
+format's associated specification may be proprietary or open.
+
+## Ingest
+
+A phase in a preservation **workflow** describing all the **preservation
+policy**-defined **tasks** performed on a **SIP** when it is received from a
+**producer**, prior to preservation. Ingest task examples include checking the
+file format against a list of acceptable formats and checking for hidden files.
+Typically, ingest concludes by finalizing the transformation of a SIP into a
+**PIP** for preservation processing by the **preservation engine**.
+
+## Intellectual entity
+
+The [PREMIS](https://www.loc.gov/standards/premis/) 3 Data Dictionary defines an
```

Comment:
The "3" looks so lonely outside the hyperlink. :pleading_face: Maybe "The [PREMIS v3 Data Dictionary](https://www.loc.gov/standards/premis/v3/premis-3-0-final.pdf) defines..."?

Reply:
:+1: